src/main/java/net/oauth/jsontoken/discovery/VerifierProviders.java

```diff
@@ -31,7 +32,7 @@
  * will use different ways to look up verification keys - for example, symmetric keys
  * will always be pre-negotiated and looked up in a local database, while public
  * verification keys can be looked up on demand), and the ask the {@link VerifierProvider}
- * to provide a {@link List<Verifier>} to check the validity of the JSON Token.
+ * to provide a {@link List <Verifier>} to check the validity of the JSON Token.
```

Comment:
accidental change?

Reply:
(also it should read `{@code List<Verifier>}`)

Reply:
Fixed! This issue will also be fixed in unrelated files in another PR.